client/webserver/locales/pl-pl.go

```diff
@@ -0,0 +1,211 @@
+	"Authorize the withdraw with your app password.": "Zatwierdź wypłatę swoim hasłem aplikacji.",
+	"Reconfigure":                 "Skonfiguruj ponownie",
+	"pw_change_instructions":      "Zmiana poniższego hasła nie powoduje zmiany hasła do Twojego oprogramowania portfela. Skorzystaj z tego formularza, aby zaktualizować klienta DEX po tym, jak zmienisz hasło do swojego oprogramowania portfela.",
+	"New Wallet Password":         "Nowe hasło portfela",
+	"pw_change_warn":              "Uwaga: Zmiana portfela podczas gdy trwają wymiany może spowodować utratę środków.",
+	"Show more options":           "Wyświetl więcej opcji",
+	"seed_implore_msg":            "Zapisz ziarno aplikacji dokładnie na kartce papieru i zachowaj jego kopię. Jeśli stracisz dostęp do tego urządzenia lub niezbędnych plików aplikacji, ziarno to umożliwi Ci przywrócenie kont DEX oraz wbudowanych portfeli. Niektóre starsze konta nie mogę być przywrócone ta metodą, i niezależenie od tego, czy konto jest nowe, czy stare, zachowanie kopii swoich kluczy w dodatku do ziarna jest zawsze dobrą praktyką.",
+	"View Application Seed":       "Wyświetl ziarno aplikacji",
+	"Remember my password":        "Zapamiętaj hasło",
+	"pw_for_seed":                 "Enter your app password to show your seed. Make sure nobody else can see your screen.",
+	"Asset":                       "Aktywo",
+	"Balance":                     "Saldo",
+	"Actions":                     "Czynności",
+	"Restoration Seed":            "Ziarno do przywrócenia",
+	"Restore from seed":           "Przywróć z ziarna",
+	"Import Account":              "Importuj konto",
+	"no_wallet":                   "brak portfela",
+	"create_a_x_wallet":           "Utwórz portfel {{.Info.Name}}", // .Info.Name will be an asset symbol like DCR
+	"dont_share":                  "Nie udostępniaj nikomu. Nie zgub go.",
+	"Show Me":                     "Pokaż",
+	"Wallet Settings":             "Ustawienia portfela",
+	"add_a_x_wallet":              `Dodaj portfel <img data-tmpl="assetLogo" class="asset-logo mx-1"> <span data-tmpl="assetName"></span>`,
+	"ready":                       "gotowy",
+	"off":                         "wyłączony",
+	"Export Trades":               "Eksportuj zlecenia wymiany",
+	"change the wallet type":      "zmień typ portfela",
+	"confirmations":               "potwierdzenia",
+	"how_reg":                     "Jak chcesz uiścić opłatę rejestracyjną?",
+	"All markets at":              "Wszystkie rynki na",
+	"pick a different asset":      "wyierz inne aktywo",
+	"Register_loudly":             "Zarejestruj!",
+	"1 Sync the Blockchain":       "1: Zsynchronizuj blockchain",
+	"Progress":                    "Postęp",
+	"remaining":                   "pozostało",
+	"2 Fund the Registration Fee": "2: Wnieś opłatę rejestracyjną",
+	"Registration fee":            "Opłata rejestracyjna",
+	"Your Deposit Address":        "Twój adres do wpłaty",
+	"add a different server":      "dodaj inny serwer",
```

Comment:
@artikozel On master there's one right between the two above:

```diff
	"Your Deposit Address":        "Your Deposit Address",
+	"Send enough for reg fee":     `Make sure you send enough to also cover network fees.`,
	"add a different server":      "add a different server",
```

@JoeGruffins posted a screenshot of this here: https://github.com/decred/dcrdex/pull/1422#issuecomment-1014572770

Reply:
Oh, @artikozel, you're welcome to just provide edits in comments on Github.  Alternatively, if you get the git business all handled, you can get a branch up to replace this PR.  Your call.  No sweat either way.

Reply:
@chappjc
"Send enough for reg fee":     `Upewnij się, że wysyłasz wystarczająco dużo środków, aby pokryć również opłaty sieciowe.`

Still fighting github. This way's dirtier, but speed is of key improtance here :)